### PR TITLE
Linkplus

### LINKED ISSUE #136: Travian plus buttons
I wanna make these work. Thoughts on bannability? Can just appeal and say you were using browser bookmarks if they ever ask?

### PR DESCRIPTION
**Pull request by Geczy**
Close #136

### CODE REVIEW

File: Plugins/Fun/Decorations/Decorations.js [hunk 1]
```diff
@@ -37,6 +38,35 @@ function Decorations() {
...
+
+		var replaceClasses = [
+			"workshop",
+			"stable",
+			"barracks",
+			"market",
+		];
+
+		var ids = [
+			21,
```
Comment (Geczy) [inline Plugins/Fun/Decorations/Decorations.js]: Would wanna use the enums to get the gid for this but didn't want to create a for loop to find a gid by name

**Comment by AleksandarDev** [comment]
Is this finished? Works nice for me.

**Comment by Geczy** [comment]
Yeah, would be optimal if I can remove the hover and the click javascript from them though

**Comment by Geczy** [comment]
merge?

**Comment by AleksandarDev** [comment]
I can't merge. Resolve conflicts and commit changes please.

**Comment by Geczy** [comment]
Try now

**Pull request merged**